src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -495,6 +498,10 @@ private static void addAllConfigKeys(ConfigDef container, ConfigDef other, Set<S
     }
   }
 
+  public int getTaskId() {
+    return taskId != null ? Integer.parseInt(taskId) : -1;
```

Comment:
can we declare this member `final` and move this logic in the constructor now? That'd be preferable if we can